.eslintrc.json

```diff
@@ -0,0 +1,57 @@
+// {
+//     "env": {
+//         "browser": true,
+//         "es2021": true
+//     },
+//     "extends": [
+//         "plugin:react/recommended",
+//         "airbnb"
+//     ],
+//     "parserOptions": {
+//         "ecmaFeatures": {
+//             "jsx": true
+//         },
+//         "ecmaVersion": "latest",
+//         "sourceType": "module"
+//     },
+//     "plugins": [
+//         "react"
+//     ],
+//     "rules": {
+//     }
+// }
+
+{
+
```

Comment:
remove commented code

Reply:
Sure. I have done git rebase and will remove those commented codes soon. However, the pre-commit that will run the tests is blocked by the setup testing but I'll add it as soon as the setup testing is merged.